### DISCUSSION TITLE
Getting 403 Forbidden: Access is denied for livewire.js in Azure

### DISCUSSION THREAD

**Discussion by fernando-moreno** [livewire/livewire#4198] [1. Help] [answered]
Hi, I have deployed my app successfully except for this 

![image](https://user-images.githubusercontent.com/70163154/141340672-f1506ba3-f19c-4aba-a313-60145ab9ede8.png)

I have published livewire assets already, livewire.js is on the right folder and livewire.config is routing correcting. Seems that just cannot be opened.

Any suggestions?

**Comment by ralphjsmit** [comment, marked as answer]
What is the file path that you're trying to load? Where is the `livewire.js` located?

Have you used the `@livewireScripts` Blade directive?

I can't see the full URL, but it looks like you're trying to load `livewire.js` from the `vendor` folder, whilst the vendor folder is not accessible from the frontend (only everything inside the `/public` folder is).

**Reply by fernando-moreno** [reply]
Hi, yes I have used the `livewireScripts `Blade directive.

It is trying to load `livewwire.js` from 

![image](https://user-images.githubusercontent.com/70163154/141348657-c373ed25-6d7b-487b-b066-52953864fd49.png)

it is searching inside the vendor folder within public folder.

Even from the browser if I recall app.css file in public folder it works. If I have move `livewire.js` to css can be accessed and routed there but it breaks all the components locations.

my `vendor/livewire/livewire.js` is in `/public`

Seems like the vendor folder in public has some read restrictions, would that make sense?

**Reply by fernando-moreno** [reply]
Managed to solve it (kind of).

This problem in IIS in the Azure Web App. No matter the setup `livewire.js` was unreachable even though it was on the public folder.

I removed the whole resource and started the web app as Linux server, installed Composer and setup Laravel and it works now.

Setting up Laravel in Azure using IIS is a bit easier but at the end works better with Linux environment. 

BTW, locally I develop on WSL.

**Reply by ralphjsmit** [reply]
That's weird! It could be indeed be that the file permissions weren't fully configured correctly. Happy to hear that it works!
